content/livesync/Setup

```diff
@@ -0,0 +1,190 @@
+product: LiveSync
+---
+
+Use these instructions to install, authenticate and instantiate the Ably Realtime Data Models SDK.
+
+h2(#authenticate). Authenticate
+
+An "API key":/auth#api-keys is required to authenticate with Ably. API keys are used either to authenticate directly with Ably using "basic authentication":/auth/basic, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+<p>The examples use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication":/auth/token. </p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
+
+API keys and tokens have a set of capabilities assigned to them that specify which operations, such as @subscribe@ or @publish@ can be performed on which resources. To use the Ably Realtime Data Models SDK, the API key requires the following "capabilities":/auth/capabilities#58: 
+
+* Publish
+* Subscribe
+* Presence
+* History
+
+h2(#install). Install
+
+The Realtime Data Models SDK is a JavaScript (TypeScript) library that enables you to create live and observable data models in your frontend application. These models remain synchronized with the realtime state of your database model. You can easily integrate this SDK into your project regardless of your frontend framework preference.
+
+Your backend publishes mutation events to Ably. The Realtime Data Models SDK updates your frontend app's local state. You can also pair the SDK with *Ably's Database Connector* to transmit transactional change events with your database mutations.
+
+The Realtime Data Models SDK is available as an "NPM module":#npm.
+
+h3(#npm1). Use NPM
+
+Install the Ably JavaScript SDK and the Realtime Data Models SDK:
+
+```
+npm install ably @ably-labs/models
+```
+
+
+h3(#npm-link). Use the NPM link
+
+Clone this repository and run @npm link@:
+
+```
+git clone git@github.com:ably-labs/models.git
+cd models
+npm link
+```
+
+From your project, link to the cloned project and build:
+
+```
+cd ./your/project
+npm link @ably-labs/models
+pushd ./node_modules/@ably-labs/models
+npm run build
+popd
+```
+
+You should now be able to import @@ably-labs/models@ in your project.
+
+h2(#instantiate). Instantiate
+
+To instantiate the Realtime Data Models SDK, create an "Ably client":/getting-started/setup?lang=javascript and pass it into the @ModelsClient@ constructor:
+
+```
+import ModelsClient from '@ably-labs/models';
+import { Realtime } from 'ably';
+
+const ably = new Realtime.Promise({ key: 'YOUR_ABLY_API_KEY' });
+const modelsClient = new ModelsClient({ ably });
+```
+
+h2(#create-model). Create a model
```

Comment:
I'd say anything below this point is part of using the SDK, rather than its setup. 

May be worth creating a ticket to add a section on the benefits of the underlying "core" client here too - but that can come later.